.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,11 @@
 <!-- What's in this pull request? -->
 Replace this paragraph with a description of your changes and rationale. Provide links to external references/discussions if appropriate.
 
-<!-- If this pull request resolves any bugs in the Swift bug tracker, provide a link: -->
-Resolves SR-NNNN.
+<!-- If this pull request resolves any GitHub issues, link them.
+For information about linking a pull request to an issue, see:
+https://docs.github.com/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue
+-->
+Resolves #NNNNN, fix apple/llvm-project#MMMMM.
 
 <!--
 Before merging this pull request, you must run the Swift continuous integration tests.
```

Comment:
Sorry to be nit-picky, but could we format multiline comments consistently? This one has a line break following the opening tag.

Reply:
No problem, done.